app/views/RoomActionsView/ActionsList/RoomItem.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { View, Text } from 'react-native';
+import { withNavigation } from 'react-navigation';
+import PropTypes from 'prop-types';
+
+import Avatar from '../../../containers/Avatar';
+import Status from '../../../containers/Status';
+import DisclosureIndicator from '../../../containers/DisclosureIndicator';
+import Touch from '../../../utils/touch';
+import RoomTypeIcon from '../../../containers/RoomTypeIcon';
+
+import { COLOR_WHITE } from '../../../constants/colors';
+import styles from './styles';
+import sharedStyles from '../../Styles';
+
+const RoomItem = ({
+	name, baseUrl, user, member, route, params, testID, navigation
+}) => {
+	const { room } = params;
+	const { navigate } = navigation;
+
+	return (
+		<Touch
+			onPress={() => navigate(route, params)}
+			underlayColor={COLOR_WHITE}
+			activeOpacity={0.5}
+			accessibilityLabel={name}
+			accessibilityTraits='button'
+			testID={testID}
+		>
+			<View style={styles.sectionItem}>
+				<Avatar
+					key='avatar'

```

Comment:
Since we're refactoring, these `key` aren't needed anymore.
They were needed when we were using array to declare components (when React.Fragment, aka `<>`, didn't exist yet)